### PR TITLE
Form Builder | Add form section query args to GQL

### PR DESCRIPTION
**Pull request by manzoorwanijk**
This PR adds query args to filter form sections in GQL schema.

### CODE REVIEW

File: core/domain/services/graphql/connection_resolvers/FormSectionConnectionResolver.php [hunk 1]
```diff
@@ -91,4 +117,25 @@ public function get_query_args(): array
...
+        $arg_mapping = [
+            'appliesTo' => 'FSC_appliesTo',
+            'belongsTo' => 'FSC_belongsTo',
+            'status'    => 'FSC_status',
+        ];
+        return $this->sanitizeWhereArgsForInputFields(
+            $where_args,
+            $arg_mapping
+        );
+    }
```
Comment (tn3rb) [inline core/domain/services/graphql/connection_resolvers/FormSectionConnectionResolver.php:140]: unless we are going to need to manipulate this array in some way, let's just roll it right into the function call and skip using a variable, ie:

```suggestion
    public function sanitizeInputFields(array $where_args): array
    {
        return $this->sanitizeWhereArgsForInputFields(
            $where_args,
            [
            	'appliesTo' => 'FSC_appliesTo',
            	'belongsTo' => 'FSC_belongsTo',
            	'status'    => 'FSC_status',
        	]
        );
    }
```

**Review by tn3rb — APPROVED** [review]
AWESOME! 👍🏻 👍🏻 

had one super minor format request/suggestion

but this code is great as is

File: core/domain/services/graphql/connection_resolvers/FormSectionConnectionResolver.php (hunk 1 above)
Comment (manzoorwanijk) [reply core/domain/services/graphql/connection_resolvers/FormSectionConnectionResolver.php:140]: Will make that change in the next PR. Thank you

Comment (manzoorwanijk) [reply core/domain/services/graphql/connection_resolvers/FormSectionConnectionResolver.php:140]: Done [here](https://github.com/eventespresso/event-espresso-core/pull/3441/files#r658090766)

Comment (tn3rb) [reply core/domain/services/graphql/connection_resolvers/FormSectionConnectionResolver.php:140]: 🙇🏻

**Pull request merged**